File: 2026-09/src/day_13/prms_120883/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(vector<string> id_pw, vector<vector<string>> db) {
    for (auto& row : db) {
        if (id_pw[0] != row[0]) continue;
        return id_pw[1] == row[1] ? "login" : "wrong pw";
    }

    return "fail";
}

File: 2026-09/src/day_13/prms_120883/Solution.java
package day_13.prms_120883;

class Solution {
    public String solution(String[] id_pw, String[][] db) {
        for (String[] row : db) {
            if (!id_pw[0].equals(row[0])) continue;
            return id_pw[1].equals(row[1]) ? "login" : "wrong pw";
        }

        return "fail";
    }
}
